x-pack/plugins/saved_objects_tagging/server/plugin.ts

```diff
@@ -54,11 +54,12 @@ export class SavedObjectTaggingPlugin
     features.registerKibanaFeature(savedObjectsTaggingFeature);
 
     if (usageCollection) {
-      const kibanaIndices = savedObjects.getAllIndices();
+      const getKibanaIndices = () =>
+        getStartServices().then(([core]) => core.savedObjects.getAllIndices());
```

Comment:
Question: should we add a `catch` and return an empty array or is that handled in `core`?

Reply:
In practice, the only reason for `getStartServices` to reject is if the server shutdowns before `start`. But I agree that it's safer, I will add a catch 👍